wrapanapi/systems/msazure.py

```diff
@@ -1130,6 +1159,21 @@ def delete_stack_by_date(self, days_old, resource_group=None):
                              result)
         return results
 
+    def delete_compute_image_by_name(self, resource_group=None, image_list=None):
+        """
+        Delete compute images by name
```

Comment:
1. I am confused by the name here. The other method that you added is named `_by_resource_group`. I would prefer naming consistency and give this method name the same suffix.

```suggestion
    def delete_compute_image_by_resource_group_(self, resource_group=None, image_list=None):
        """
        Delete compute images by resource group
```

2. Shouldn't `image_list` be a named argument? Does it make sense to actually do anything if the `image_list` is empty?

Reply:
`2.` is non-blocking

`1.` is blocking comment from my PoV
My intention is not to nitpick, but in 63499f9 you have changed `delete_compute_image_by_name` to `delete_compute_image_using_name`, but since the method deletes compute images that are filtered by **resource_group**, I would still insist on naming the method `delete_compute_image_by_resource_group`